### DISCUSSION TITLE
Port / Pin selection for multiple spindle

### DISCUSSION THREAD

**Discussion by samitray** [grblHAL/core#529] [Q&A]
For multiple spindle N_SPINDLE  2, I have some queries regarding pin and port selection for the second spindle pwm pin. most of the map files have spindle pwm on PA8 which is TIM1_CH1, so i assume that for the second spindle TIM2,TIM3,TIM9,TIM11 is allowed as per pwm_enable in /src/pwm.c and the driver code configures alternate function to set PWM output on the pin,  however i am not clear if ONLY  CH1 (of the corresponding timers in pwm_enable) is allowed or other channels can be used.

**Comment by terjeio** [comment]
[Other channels can be used](https://github.com/grblHAL/STM32F4xx/blob/7b5f81c7bf5a56c06d72798ca787316b79a4500a/Src/pwm.c#L30-L32) - CH = channel.
